feat: update snippets documentation [sc-9215] (#467)
checkly/checklyhq.com: site/content/docs/api-checks/setup-teardown-scripts.md

![setup and teardown script for http requests with snippets](/docs/images/api-checks/setup-teardown-snippet.png)

> It's also possible to import snippets from your setup or teardown script. For more information see the [documentation page on snippets](/docs/snippets). This can be particularly useful if setup and teardown scripts are similar across checks, but still require some customization.

Reviewer: ```suggestion
It's also possible to import snippets from your setup or teardown scripts. This can be valuable if setup and teardown scripts are similar across checks, but still require some customization. Find more information in the [snippet documentation](/docs/snippets). 
```
